ci-operator/config/openshift/csi-driver-shared-resource/openshift-csi-driver-shared-resource-release-4.12.yaml

```diff
@@ -15,16 +15,23 @@ binary_build_commands: make build
 build_root:
   from_repository: true
 images:
-- dockerfile_path: Dockerfile
+- dockerfile_path: images/Dockerfile
   from: base
   inputs:
     ocp_builder_rhel-8-golang-1.17-openshift-4.11:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
   to: csi-driver-shared-resource
-- dockerfile_path: Dockerfile.mustgather
+- dockerfile_path: images/Dockerfile.mustgather
   from: must_gather
   to: csi-driver-shared-resource-mustgather
+- dockerfile_path: images/Dockerfile.webhook
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.17-openshift-4.11:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
```

Comment:
yep double checked and there is no `openshift-4.12` yet so this is correct as I understand it